geometry_msgs/Pose pose
string obj_name
